lib/stylesheet/importer.rb

```diff
@@ -40,6 +40,32 @@ def self.register_imports!
         end
       end
 
+      register_import "fonts" do
+        contents = +""
+
+        DiscourseFonts.fonts.each do |font|
+          if font[:variants].present?
+            font[:variants].each do |variant|
+              contents << <<~EOF
+                @font-face {
+                  font-family: #{font[:name]};
+                  src: asset-url("/fonts/#{variant[:filename]}?v=#{DiscourseFonts::VERSION}") format("#{variant[:format]}");
+                  font-weight: #{variant[:weight]};
+                }
+              EOF
+            end
+          end
+
+          contents << <<~EOF
```

Comment:
Having a class definition per font makes sense for the wizard, but it's not necessary for the regular app. I think you can just do 

```
:root {
  --font-family: #{SiteSetting.base_font};
}
```

in the regular app. That also allows themes to more easily override the base font. Doing this introduces a small caching issue, the stylesheet cache would need to be cleared every time the font site setting is updated.

Reply:
It is what I am using now and I introduced two imports to achieve that: `font` and `wizard_fonts`.